ai/orchestrators/models-config.mdx

```diff
@@ -27,6 +27,11 @@ currently **recommended** models and their respective prices.
     "model_id": "ByteDance/SDXL-Lightning",
     "price_per_unit": 4768371
   },
+  {
+    "pipeline": "upscaling",
```

Comment:
```suggestion
    "pipeline": "upscale",
```
The correct name for the pipeline is `upscale`

Reply:
Ah thanks for catching that one!